src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+    class="form__big"
+    action="#"
+    method="post"
+    data-qa="big"
+  >
```

Comment:
```suggestion
    <form
      class="form__big"
      action="#"
      method="post"
      data-qa="big"
    >
```

Reply:
I've written about it last during last review

Reply:
Sorry, I misunderstood